pom.xml

```diff
@@ -1929,7 +1929,7 @@
             <dependency>
                 <groupId>com.github.luben</groupId>
                 <artifactId>zstd-jni</artifactId>
-                <version>1.5.0-4</version>
+                <version>1.5.2-2</version>
```

Comment:
Did we compare the GC activity with the new version ?
I remember there was a UER miss previously related to zstd version bump. (T71450283)

Reply:
Yes, we compared both young GC and old GC, and there were no difference between the 2 versions.